src/utils/request.rs

```diff
@@ -15,5 +17,6 @@ pub fn build_reqwest_client(
     Ok(reqwest::ClientBuilder::new()
         .add_root_certificate(cert)
         .default_headers(headers)
+        .pool_idle_timeout(Some(Duration::from_secs(5 * 60)))
```

Comment:
This is here on purpose?

Reply:
This seems to be for event stream in the in_game.rs module, but it doesn't actually use the polling rate set there?

Reply:
I initially added for my reqwest vs hyper benchmarks to make sure the tcp connection doesnt get restarted during the test. But the default of 90s should be fine so ill remove it